lib/logthrdestdrv.c

```diff
@@ -32,8 +32,8 @@ log_threaded_dest_driver_format_seqnum_for_persist(LogThrDestDriver *self)
 {
   static gchar persist_name[256];
```

Comment:
note : maybe it is time to using a `GString` instead of the static buffer?
(or, at least we should tell the user that the persist name length has a max length...).